mu4e-dashboard.el

```diff
@@ -53,7 +53,7 @@
 
 
 (defun mu4e-dashboard-follow-mu4e-link (path)
```

Comment:
Do we rename this one too then ? `mu4e-dashboard-follow-mu-link`

Reply:
Probably makes sense... although if following best practice if this is not something we expect users to call directly we could -- after the prefix, for example mu4e-dashboard--follow-link which indicates that it's not a "public" function (in so much as all Emacs functions are actually public in the global namespace) and people are less likely too complain if you change/re-factor them to keep your internals clean.